pkg/reconciler/testing/broker.go

```diff
@@ -111,6 +111,24 @@ func WithBrokerReady(b *v1alpha1.Broker) {
 	b.Status = *v1alpha1.TestHelper.ReadyBrokerStatus()
 }
 
+func WithDeprecatedStatus(b *v1alpha1.Broker) {
+	dc := apis.Condition{
+		Type:     "Deprecated",
+		Reason:   "SingleTenantChannelBrokerDeprecated",
+		Status:   corev1.ConditionTrue,
+		Severity: apis.ConditionSeverityWarning,
+		Message:  "Single Tenant Channel Brokers are deprecated and will be removed in release 0.16. Use Multi Tenant Channel Brokers instead.",
```

Comment:
Will consumers of this be thinking in terms of eventing releases? or serverless releases?

Reply:
How about we just remove any versioning, and just indicate it will be deleted in a future version, @lberk  ?